packages/react-text/src/components/Text/Text.tsx

```diff
@@ -8,7 +8,7 @@ import { useTextStyles } from './useTextStyles';
  * Typography and styling abstraction component used to ensure consistency of text.
  */
 export const Text = React.forwardRef<HTMLElement, TextProps>((props, ref) => {
-  const state = useText(props, ref);
+  const state = useText(props, ref, { as: 'span' });
```

Comment:
```suggestion
  const state = useText(props, ref);
```

Can we move this to `useText.ts` instead?

```diff
  const state = mergeProps(
    {
      ref,
+     as: 'span',
    },
    defaultProps,
    props,
  );
```

This will be more clearer for the migration when we will apply #18642.

Reply:
Thanks for the suggestion, implemented!